### ISSUE TITLE
[orc8r] No association in DB is created while adding apn_resources via API

### ISSUE THREAD

**Issue by berezovskyi-oleksandr** [magma/magma#3149]
### Your Environment

- **Version:** 1.2 (built from latest 1.2 branch)
- **Affected Component:** Orchestrator
- **Deployment Environment:** AWS

### Describe the Issue

I want to setup Multi-APN IP allocation according to [the documentation](https://magma.github.io/magma/docs/howtos/config_agw_bridged#experimental-multi-apn-ip-allocation).

I added next section via API:
```
  "apn_resources": {
    "internet": {
      "apn_name": "internet",
      "gateway_ip": "192.168.10.1",
      "gateway_mac": "e0:63:da:22:47:21",
      "id": "internet_apn_resource",
      "vlan_id": 13
    },
    "management": {
      "apn_name": "management",
      "gateway_ip": "192.168.9.1",
      "gateway_mac": "e0:63:da:22:47:21",
      "id": "management_apn_resource",
      "vlan_id": 12
    }
  }
```

I've got 204 in response, but after I make GET request, `apn_resources` section is empty.
This leads to the fact, that no "apn_resources" configuration was populated to the AGW (I've verified that by getting into subscribers.db and deserializing protobuf message).

I went into orc8r db and performed an INSERT into `cfg_assocs` table to add two associations between entity with type `cellular_gateway` and entities with type `apn_resource`.
After this opeation, GET request started returning values with `apn_resources` section filled with values. Data was further propagated to `subscribers.db` at AGW and traffic started working as desired. 

I believe, there is some problem in logic, that implements save `apn_resource` entities into the database and further construction of association between `apn_resource` entities and `cellular_gateway` entities.

**Comment by hcgatewood** [comment]
Taking a look at this. @berezovskyi-oleksandr can you include the 1.2 branch commit you built. We've been backporting some recent apn_resources fixes to 1.2 so we may have already fixed the issue

**Comment by berezovskyi-oleksandr** [comment]
@hcgatewood , thanks for your response. I've checked the orc8r image and I believe that it was 94b202d7d commit.

Today, as @pshelar suggested in Slack, I've re-deployed orc8r on top of latest 1.3 branch with commit 54ed1bd6b.

I've re-checked everything with latest 1.3 and the situation is next:
1) now there is no "apn_resources" section once there is no associated "apn_resource";
2) there is still no assocation being created during adding new "apn_resource"
3) once I add association manually throught the DB, the "apn_resources" section appeared with the associated record, and `subscibers.db` was updated at AGW.
